no-op/src/main/kotlin/com/redmadrobot/debug/noop/accounts/AccountsPlugin.kt

```diff
@@ -1,7 +1,7 @@
-package com.redmadrobot.account_plugin.plugin
+package com.redmadrobot.debug.accounts.plugin
```

Comment:
Hm, I think it will be better to change packages of all plugins to structure like:
```suggestion
package com.redmadrobot.debug.plugin.accounts
```
To gather all plugins in common package